webui/react/src/components/kit/LineChart.tsx

```diff
@@ -70,6 +72,8 @@ export const LineChart: React.FC<Props> = ({
   xAxis = XAxisDomain.Batches,
   xLabel,
   yLabel,
+  xTickValues = undefined,
+  yTickValues = undefined,
```

Comment:
these allows us to format axis tick values

Reply:
wouldn't undefined already be the default value for these, without the `= undefined`?

Reply:
Does it make sense to send `xTickValues` and `yTickValues` from the SystemMetricChart, if we probably want this for other charts with a time x-axis?